src/sentry/api/helpers/events.py

```diff
@@ -56,6 +56,8 @@ def get_query_builder_for_group(
     dataset = Dataset.Events
     if group.issue_category == GroupCategory.PERFORMANCE:
         dataset = Dataset.Transactions
+    elif not group.issue_category == GroupCategory.ERROR:
```

Comment:
Might be worth making a mapping of issue category to dataset?

Reply:
yeah, could also reverse the logic a bit so it's two positive `if` statements which reads a little cleaner. not sure we want to make `Dataset.IssuePlatform` the default though

Reply:
I think that makes sense, it's kinda the fallback option.